content/en/account_management/org_settings/custom_landing.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Custom Organization Landing Page
+kind: documentation
+---
+
+## Overview
+
+The Datadog organization landing page is the first page your users see when they log on to Datadog or navigate to the Datadog root page. Datadog sets a default landing page for your organization. If you use APM, Datadog sets the APM root as the landing page. If you don’t use APM, then the list of dashboards is the default landing view.
```

Comment:
`list of dashboards is the default landing view.` inconsistent with "landing page" above?

Reply:
I was trying to add some variety in wording. But after reading your comment, I see that the result is confusing. I changed that last "view" to be "page" like the other sentences.